hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/TestWriteCopyOnWrite.java

```diff
@@ -540,9 +541,7 @@ public void testWriteMultiWriterInvolved() throws Exception {
         .checkWrittenData(EXPECTED3, 1);
     // step to commit the 2nd txn, should throw exception
     // for concurrent modification of same fileGroups
-    pipeline1.checkpoint(1)
-        .assertNextEvent()
-        .checkpointCompleteThrows(1, HoodieWriteConflictException.class, "Cannot resolve conflicts");
+    testConcurrentCommit(pipeline1);
```

Comment:
Can we add new tests instead of ammending existing one?

Reply:
The behavior of existed test cases (`testWriteMultiWriterInvolved` and `testWriteMultiWriterPartialOverlapping`) change after support non-blocking multiple writers. So those two tests need to be updated.
I also add some new tests.

Reply:
Thanks